vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,59 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix gmps: <http://linked.data.gov.au/def/qld-resource-permit-status/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/qld-resource-permit-status> a owl:Ontology .
+
+gmps:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-09-13"^^xsd:date ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:source "https://myminesonline.business.qld.gov.au" ;
+    skos:definition "The stage in the life cycle of a Queensland Resource Permit."@en ;
+    skos:hasTopConcept gmps:Abandoned,
+        gmps:abandoned,
+        gmps:application,
+        gmps:granted,
+        gmps:non-current,
+        gmps:unsuccessful ;
+    skos:prefLabel "Queensland Resource Permit Status"@en .
+
+gmps:abandoned a skos:Concept ;
+    skos:altLabel "Discarded"@en ;
+    skos:definition "An application for the permit that has been abandoned or discarded by the applicant."@en ;
```

Comment:
'An application for a permit that has been abandoned or discarded by the applicant'